src/taskutil.cc

```diff
@@ -95,3 +114,93 @@ bool TaskExecuteAndWait(std::string execpath, std::string *pOutput) {
     return retval;
 }
 
+/*
+int system(const char * command) {
+    if (command == nullptr)
+        return 0;
+    bool bRet = WindowsTaskExecute(command, nullptr, true);
+    return 0;//fix this //WINDOWS-TEMP
+}
+
+*/
+
+
+//see https://msdn.microsoft.com/en-us/library/ms686852(v=VS.85).aspx
+
+int CountProcessThreads(DWORD id) {
+    HANDLE hThreadSnap = INVALID_HANDLE_VALUE;
+    BOOL  retval = true;
+    PROCESSENTRY32 pe32 = { 0 };
+    int count = -1;
+    pe32.dwSize = sizeof(PROCESSENTRY32);
+
+    hThreadSnap = CreateToolhelp32Snapshot(TH32CS_SNAPALL, 0);
```

Comment:
Could not find function to get the same.  
See https://msdn.microsoft.com/en-us/library/ms686852(v=VS.85).aspx
Adding error checking.